src/include/ceph_fs.h

```diff
@@ -62,12 +62,6 @@ struct ceph_dir_layout {
 	__u32  dl_unused3;
 } __attribute__ ((packed));
 
-/* crypto algorithms */
-#define CEPH_CRYPTO_NONE 0x0
-#define CEPH_CRYPTO_AES  0x1
-
-#define CEPH_AES_IV "cephsageyudagreg"
-
```

Comment:
i'm guessing this needs to stay here for the kernel client, which has a copy of this header at https://github.com/ceph/ceph-client/blob/for-linus/include/linux/ceph/ceph_fs.h

please add the new `CEPH_CRYPTO_AES_` defines here instead of auth/Crypto.h